src/native/managed/cdacreader/src/Legacy/SOSDacImpl.cs

```diff
@@ -405,7 +405,8 @@ int ISOSDacInterface.GetMethodDescData(ulong methodDesc, ulong ip, DacpMethodDes
                 Debug.Assert(data->MDToken == dataLocal.MDToken);
                 Debug.Assert(data->GCInfo == dataLocal.GCInfo);
                 Debug.Assert(data->GCStressCodeCopy == dataLocal.GCStressCodeCopy);
-                Debug.Assert(data->managedDynamicMethodObject == dataLocal.managedDynamicMethodObject);
+                // managedDynamicMethodObject is not currently populated by the cDAC API and may differ from legacyImpl.
```

Comment:
Where is the managedDynamicMethodObject field actually zero'ed?  I don't see it anywhere. I'm not sure if you should  rely on the client to zero it (like in dacprivate.h).  It may not be enough i.e. CLRMD doesn't use that include file.

Reply:
I see. I wasn't sure, but that makes sense. I'll zero out the field above where it would have been set.